certified-connectors/Jupyrest/apiDefinition.swagger.json

```diff
@@ -0,0 +1,294 @@
+        "summary": "Run a notebook",
+        "tags": [
+          "JupyrestConnector"
+        ],
+        "operationId": "NotebookExecution",
+        "deprecated": false,
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "disableRedirect",
+            "in": "query",
+            "type": "boolean",
+            "default": true,
+            "required": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "output",
+            "in": "query",
+            "type": "boolean",
+            "default": true,
+            "required": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "html",
+            "in": "query",
+            "type": "boolean",
+            "default": true,
+            "required": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "report",
+            "in": "query",
+            "type": "boolean",
+            "default": false,
+            "required": false,
+            "x-ms-visibility": "important",
+            "x-ms-summary": "report mode",
+            "description": "If true, return report version of execution results (without notebook command cells)."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Notebook input parameters.",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "notebook": {
+                  "type": "string",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetNotebooks",
+                    "value-path": "notebook",
+                    "value-title": "notebook"
+                  }
+                },
+                "parameters": {
+                  "type": "object",
+                  "x-ms-dynamic-schema": {
+                    "operationId": "GetNotebookMetadata",
+                    "value-path": "input",
+                    "parameters": {
+                      "notebook": {
+                        "parameter": "notebook"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-properties": {
+                    "operationId": "GetNotebookMetadata",
+                    "itemValuePath": "input",
+                    "parameters": {
+                      "notebook": {
+                        "parameterReference": "parameters/notebook"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/NotebookResponse"
+            },
+            "headers": {}
```

Comment:
@litiany23 Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
done